addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -54,9 +54,11 @@ def action_send_sms(self):
                     _('Test SMS successfully sent to %s', sent_sms.get('res_id')))
             elif sent_sms.get('state'):
                 notification_messages.append(
-                    _('Test SMS could not be sent to %s: %s',
-                    sent_sms.get('res_id'),
-                    error_messages.get(sent_sms['state'], _("An error occurred.")))
+                    _(
+                        "Test SMS could not be sent to %(destination)s: %(state)s",
+                        destination=sent_sms.get("res_id"),
+                        state=error_messages.get(sent_sms["state"], _("An error occurred.")),
+                    )
```

Comment:
Nitpicking because it is a new diff. Actually python-side we only use double quote for redacted text and we use single-quote for all technical strings. So technically it should be `'res_id'` and `'state'`. I would had love to use this PR also to fix all those quotes but I guess it is too late now, also it really has no other value than to keep me quiet on reviews x) Just for my own sanity, don't break more quotes plz x)